python/lsst/analysis/tools/atools/calibQuantityProfile.py

```diff
@@ -90,16 +91,19 @@ class SingleValueRepacker(KeyedDataAction):
 
     def __call__(self, data: KeyedData, **kwargs) -> KeyedData:
         repackedData = {}
-        uniquePanelKeys = list(set(data[self.panelKey]))
+        good = ~data[self.panelKey].mask
```

Comment:
Unfortunately, I think (based on your jenkins failure), that you need to see if `data` is a numpy masked-array, and do the new thing, otherwise, default to the existing code.  Hopefully that's not too difficult, and I'll re-check next week.